code/modules/mob/living/carbon/damage_procs.dm

```diff
@@ -87,6 +87,8 @@
 			blood_volume -= 5*amount
 		else
 			blood_volume -= amount
+	if(HAS_TRAIT(src, TRAIT_TOXIMMUNE)) //Prevents toxin damage, but not healing
```

Comment:
is this an undocumented change?

Reply:
added later as a response to comment concerns, i forgot to add it to the PR desc